packages/opentelemetry-node/docs/get-started.asciidoc

```diff
@@ -0,0 +1,209 @@
+sending data to Elastic.
+
+////
+Are there any dependencies that need to be installed in addition to the distro?
+
+I think the answer here is no because the dependencies are already defined in the
+distro package so the user shouldn't have to individually install any other packages.
+If that's true, delete this comment.
+
+[discrete]
+[[get-started-add-dependencies]]
+==== Add dependencies
+////
+
+////
+✅ Provide _minimal_ configuration/setup
+////
+[discrete]
+[[get-started-configure]]
+==== Configure the distro
+
+////
+✅ Step-by-step instructions
+////
+To configure the distro, at a minimum you'll need the deployment's OTLP endpoint and
+authorization data to set the appropriate `OTLP_*` environment variables:
+
+include::configure.asciidoc[tag=minimal-options]
+
+When sending data to Elastic, you can find the values of these variables in Kibana's APM tutorial.
+In Kibana:
+
+. Search for _APM Tutorial_
+. Scroll down to the _APM Agents_ section and select the *OpenTelemetry* tab
+. The appropriate values for `OTEL_EXPORTER_OTLP_ENDPOINT` and
+`OTEL_EXPORTER_OTLP_HEADERS` are shown there
++
+For example:
++
+[source,sh]
+----
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://my-deployment.apm.us-west1.gcp.cloud.es.io
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer P....l"
+----
+
+image::img/otlp-endpoint-settings.png[Kibana's APM tutorial showing OTel settings]
+
+For more information on all the available configuration options, refer to <<configure>>.
+
+////
+✅ Start sending data to Elastic
+////
+[discrete]
+[[get-started-initialization]]
+==== Initialize the distro
+
+For the distro to automatically instrument modules used by your Node.js service,
+it must be started before you `require` your service code's dependencies --
+for example, before `express` or `http` are loaded.
+
+////
+✅ Step-by-step instructions
+////
+The recommended way to get the
+distro started is by using the `-r, --require` Node.js
+https://nodejs.org/api/cli.html#-r---require-module[CLI option]:
+
+[source,sh]
+----
+node --require @elastic/opentelemetry-node my-service.js
+----
+
+The distro will automatically instrument popular modules (listed in <<supported-technologies>>)
+used by your service, and send traces, metrics, and logs telemetry data (using
+OTLP) to your configured observability backend.
+
+// TODO: link to a reference section on other ways to start the distro once we have those docs.
+
+////
+TO DO:
+What should they expect to see after setting up and running the distro?
+How do they know it worked as expected?
+////
+[discrete]
+[[get-started-asp-net-core-view-traces]]
+=== View traces
+
+To view traces, you can use the Elastic APM UI:
+
+////
+TO DO: Step-by-step instructions
+////
+
+////
+Optional: Add an image
+////
+
+////
+TO DO:
+What should they do next?
+////
+[discrete]
+[[get-started-next-steps]]
+=== Next steps
+
+////
+// Examples:
+//
+// See all configuration options
+// Learn more about interpreting data
+// Other
+////
```

Comment:
Do we have some general Elastic Observability docs (with an eye to OpenTelemetry) that the user can be directed to for further reading?

Eventually we should have an "advanced usage" doc for the Node.js distro, and perhaps tech-specific docs for things like AWS Lambda or other FaaS systems, particular popular web frameworks (Express, Fastify, etc.). But for now we have a little.

Perhaps this is a chance to again point out the issue tracker and/or Discuss forum while this distro is pre-GA.